engo_glfw_test.go

```diff
@@ -0,0 +1,29 @@
+//+build !netgo,!android
+
+package engo
+
+import (
+	"testing"
+
+	"github.com/github.com/stretchr/testify/assert"
+)
+
+func TestGameSize(t *testing.T) {
```

Comment:
That's the point of the tests to make sure the given function is doing what it's supposed to do. Same for all below.